apps/admin-gui-e2e/src/support/commands.ts

```diff
@@ -23,3 +23,13 @@
 //
 // -- This will overwrite an existing command --
 // Cypress.Commands.overwrite("visit", (originalFn, url, options) => { ... })
+
+Cypress.Commands.add('login', (envRole: string, baPrincipalName: string) => {
+  envRole = envRole ? '_' + envRole : envRole;
+
+  sessionStorage.setItem('baPrincipal', `{"name": "${baPrincipalName}"`);
+  sessionStorage.setItem('basicUsername', Cypress.env('BA_USERNAME' + envRole) as string);
```

Comment:
It should be achievable like this (the same also for the password).
```suggestion
  sessionStorage.setItem('basicUsername', String(Cypress.env('BA_USERNAME' + envRole)));
```